content/docs/2.0/scalers/azure-log-analytics.md

```diff
@@ -0,0 +1,150 @@
+
+A data types of your query result should be: real, int or long. Other data types are not supported. Later, during runtime, your data will be converted to int64.
+
+Be careful with setting up "pollingInterval" and long running queries. Test your query before.
+
+Example query to get MetricValue and Threshold based on CPU usage and Limits, defined for the pod.
+```
+let AppName = "web";
+let ClusterName = "demo-cluster";
+let AvgDuration = ago(10m);
+let ThresholdCoefficient = 0.8;
+Perf
+| where InstanceName contains AppName
+| where InstanceName contains ClusterName
+| where CounterName == "cpuUsageNanoCores"
+| where TimeGenerated > AvgDuration
+| extend AppName = substring(InstanceName, indexof((InstanceName), "/", 0, -1, 10) + 1)
+| summarize MetricValue=round(avg(CounterValue)) by CounterName, AppName
+| join (Perf 
+        | where InstanceName contains AppName
+        | where InstanceName contains ClusterName
+        | where CounterName == "cpuLimitNanoCores"
+        | where TimeGenerated > AvgDuration
+        | extend AppName = substring(InstanceName, indexof((InstanceName), "/", 0, -1, 10) + 1)
+        | summarize arg_max(TimeGenerated, *) by AppName, CounterName
+        | project Limit = CounterValue, TimeGenerated, CounterPath, AppName)
+        on AppName
+| project MetricValue, Threshold = Limit * ThresholdCoefficient
+```
+
+You can convert your query in JSON escaped string using [this](https://www.freeformatter.com/json-escape.html) tool and check yaml specific escapes.
+
+**Scaler limitations (READ CAREFULLY):**
+
+- As Log Analytics Scaler using Service Principal to authorize REST API requests, you should be aware of throttling. You should take to account the following AAD limit: 200 requests per 30 seconds. Each Log Analytics Scaler request's Access Token twice per pooling interval (one by Keda Operator and one by Keda Metrics Server). So, theoretical maximum number of Log Analytics scalers, created with the same Service Principal for 30sec pooling interval can be 100. Read more about throttling: [here](https://dev.applicationinsights.io/documentation/Authorization/Rate-limits), [here](https://docs.microsoft.com/en-us/previous-versions/azure/ad/graph/howto/azure-ad-graph-api-throttling#what-is-throttling) and [here](https://docs.microsoft.com/en-us/azure/active-directory/users-groups-roles/directory-service-limits-restrictions)
+- As it was mentioned before, you can define a threshold using query (2d cell of query result will be interpret as threshold). Be aware! Threshold from query result will be set only once, during scaler creation. So, if your query will return different threshold values during runtime, they will not be propagated to HPA target.
+  
+### Authentication Parameters
+
+ You can use `TriggerAuthentication` CRD to configure the `tenantID`, `clientID`, `clientSecret` and `workspaceID` to connect to Log Analytics API.
+
+**Service Principal based authentication:**

```

Comment:
Can you align with other scalers here please?

https://keda.sh/docs/1.5/scalers/azure-monitor/#authentication-parameters

For example, provide more fluent sentences (example: `appId` -> `ID of Azure AD Application`)

Reply:
Done in last commit

Reply:
Thanks!